### PR TITLE
Edited UG 

### PR DESCRIPTION
**Pull request by Bisceto**
Edited UG for second review:

-Added Table of contents
-Added Command Summary
-Edited style of UG to make it consistent
-Added screenshots

Yet to add:
-Further navigability in UG
-Page formatting when converting to pdf
-Additional screenshots for certain features

### CODE REVIEW

**Comment by Bisceto** [comment]
Additional commits:
- Edited Find feature to check KEYWORD as a substring instead of exactly matching KEYWORD.
- Edited AddCommand display message, removed tags.

**Review by Jnjy — CHANGES_REQUESTED** [review]
Minor changes, other than that LGTM!

File: src/test/java/seedu/wife/logic/commands/foodcommands/FindCommandTest.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void equals() {
 
     @Test
     public void execute_zeroKeywords_noPersonFound() {
```
Comment (Jnjy) [inline src/test/java/seedu/wife/logic/commands/foodcommands/FindCommandTest.java:55]: Can change this to Food? Same as line 66

**Review by Jnjy — CHANGES_REQUESTED** [review]
Minor changes, other than that LGTM!

**Pull request merged**